test/mock-objects/mock-calculation.js

```diff
@@ -2,7 +2,8 @@ const rawCalculationData = {
   calculationId: 1,

```

Comment:
There is already a `mock-calculation` object defined in the test folder, can we not use that?

Reply:
moved to ticket https://eaflood.atlassian.net/browse/PAY-2364